Tools/CISupport/ews-build/steps.py

```diff
@@ -7040,31 +7040,39 @@ class ParseStaticAnalyzerResultsWithoutChange(ParseStaticAnalyzerResults):
     scan_build_output = SCAN_BUILD_OUTPUT_DIR + '-baseline'
 
 
-class FindUnexpectedStaticAnalyzerResults(shell.ShellCommandNewStyle):
+class FindUnexpectedStaticAnalyzerResults(shell.ShellCommandNewStyle, AddToLogMixin):
     name = 'find-unexpected-static-analyzer-results'
     description = ['finding unexpected static analyzer results']
     descriptionDone = ['found unexpected static analyzer results']
     result_message = ''
+    results_db_log_name = 'results-db'
+    jsonFileName = f'{SCAN_BUILD_OUTPUT_DIR}/unexpected_results.json'
+    logfiles = {'json': jsonFileName}
+    suite = 'safer-cpp-checks'
 
-    def __init__(self, expectations=False, **kwargs):
+    def __init__(self, expectations=False, filtered_changes=False, **kwargs):
```

Comment:
I guess it could be named `was_filtered` or something to that effect? Essentially I want to track whether or not going through results db actually changed the results.